Dynamic/Types/MultiChoiceTrait.php

```diff
@@ -22,18 +22,28 @@
      * Returns options for multichoice form type like select, multiple select, radio or checkboxes.
      *
      * @param FormFieldTranslation $translation
+     * @param bool $required
      * @param bool $expanded
      * @param bool $multiple
      *
      * @return array
      */
-    public function getChoiceOptions(FormFieldTranslation $translation, $expanded = false, $multiple = false)
-    {
+    private function getChoiceOptions(
+        FormFieldTranslation $translation,
+        $required = false,
+        $expanded = false,
+        $multiple = false
+    ) {
         $options = [];
         if ($translation) {
             // Placeholder.
             $options['placeholder'] = $translation->getPlaceholder();
 
+            if (!$options['placeholder'] && !$multiple && !$required) {
+                $options['placeholder'] = 'sulu_form.no_choice';
+                $options['translation_domain'] = 'messages';
```

Comment:
why does this options depends on multiple and required?

Reply:
`multicheckbox` and `multiselect` does not render the placeholder only Radio Button or Single Select will render the placeholder as empty field and this should only happen when not required.